### DISCUSSION TITLE
Combobox with command and Get()

### DISCUSSION THREAD

**Discussion by btcmatos** [TomSchimansky/CustomTkinter#2050] [Q&A]
I had a whole code done with tkinter and now i'm trying to adapt it to ctk to get better GUI.
One of the issues i'm having is with the combobox command, on my code i have to call a method from the combobox item change while reading the actual selected item.
What i'm getting is this (easier to show :) )

_code:
citynamebox = ctk.CTkComboBox(master=sw, state="readonly", justify='center',values=getacctname(acctbox.get()), command=print(citynamebox.get()))

error:
    citynamebox = ctk.CTkComboBox(master=sw, state="readonly", justify='center',values=getacctname(acctbox.get()), command=print(citynamebox.get()))
                                                                                                                                 ^^^^^^^^^^^
NameError: name 'citynamebox' is not defined_

I had this working the exact same way in tkinter and i need to get it to work without massive changes to the code pls !!

**Comment by zSynctic** [comment]
Double check the spelling there might be a mistake there, also having a function in command= with parenthesis makes it run at the start of the program
